packages/pwa/app/utils/locale.js

```diff
@@ -48,8 +48,11 @@ export const loadLocaleData = async (locale) => {
  */
 export const getLocaleConfig = async ({getUserPreferredLocales} = {}) => {
     const preferredLocales = getUserPreferredLocales ? getUserPreferredLocales() : [DEFAULT_LOCALE]
-    const targetLocale = getTargetLocale(preferredLocales, SUPPORTED_LOCALES, DEFAULT_LOCALE)
-    const messages = await loadLocaleData(targetLocale)
+    const targetLocale = whichLocaleToLoad(preferredLocales, SUPPORTED_LOCALES, DEFAULT_LOCALE)
+
+    const messages = await loadLocaleData(
+        typeof window === 'undefined' ? process.env.PSEUDOLOCALE || targetLocale : targetLocale
```

Comment:
Pseudo locale is now isolated to this part of the code only. This way the app could still have the target locale being English, French, etc. while the actual translations being dynamically loaded to use the pseudo locale.

Reply:
There is something that is a little off and it might be just a naming thing. 

We have an environment variable called "PSEUDOLOCALE" that when set to a value (a short code) it will use those messages over the target locale determined on the line above it. This is fine, and we are specifically doing this to allow us to control what messages to load, the pseudo locale message, and not effect other things like the locale selector.

To me this sounds a lot more like a locale override functionality. One reason that makes the current solution weird is that we have an environment variable called "PSEUDOLOCALE" but technically you can set it to `en-GB`. If you want to specifically use the word "pseudo" in your environment variable, then maybe something like `USE_PSEUDO_LOCAL_MESSAGES=true` would be more descriptive to what is actually happening. Or if you want to be a little more generic you can do something like `LOCALE_MESSAGES_OVERRIDE=en-XB`

Reply:
Good point. I'll use the boolean value then. It's what I actually wanted: narrow down the use case to pseudo locale only. And yup, it doesn't make sense to allow setting the pseudo locale value, since there's only 1 accepted value for it.